data/HarshPanchal18.json

```
@@ -0,0 +1,63 @@
+{
+  "name": "Harsh Panchal",
+  "type": "personal",
+  "bio": "Hi, I'm HARSH!\
+I'm a Software engineer in growth mode, lifelong-learner, and reader.\
+Native Android Programming, Backend, observing and developing tools always excite me.\
+Additionally, I enjoy listening to music, reading books, and acquiring new knowledge.",
```

```suggestion
  "bio": "Hi, I'm HARSH!\nI'm a Software engineer in growth mode, lifelong-learner, and reader.\nNative Android Programming, Backend, observing and developing tools always excite me.\nAdditionally, I enjoy listening to music, reading books, and acquiring new knowledge.",
```
Please use `\n` for new line